packages/xo-server-sdn-controller/README.md

```diff
@@ -11,6 +11,7 @@ For installing XO and the plugins from the sources, please take a look at [the d
 ### Network creation
 
 In the network creation view, select a `pool` and `Private network`.
+Select other pools to add them to the network if wanted.
 Create the network.
```

Comment:
If these represent steps, maybe it would be best to use a list.

Take a look at the markdown preview.